tests/cpp/fbc-mangle.bas

```diff
@@ -0,0 +1,215 @@
+' TEST_MODE : MULTI_MODULE_TEST
+
+'' test mapping of basic data types between fbc and g++
+'' with c++ name mangling
+
+#ifdef __FB_64BIT__
+
+	type cxxint as long
+	type cxxlongint as integer
+
+	/' !!!
+		cxxlongint will never work on win64, fbc custom mangles INTEGER and there is 
```

Comment:
Does that mean that the test will fail to compile on win64? If so we should probably disable it with #ifdefs etc. for that case.

Reply:
Ah ok, that must be why that part is commented out below. I suppose it could use `#if (not defined(__FB_64BIT__)) or defined(__FB_UNIX__)` or similar.